### PR TITLE
update cisco_umbrella readme

### PR DESCRIPTION
**Pull request by cherryleaf**
Added link to Cisco's documentation

Type of change

- Enhancement

## What does this PR do?

Added link to Cisco's documentation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by bmorelli25 — CHANGES_REQUESTED** [review]
@cherryleaf, there are new changes to this integration. Please merge with main and update the changelog and manifest accordingly.

File: packages/cisco_umbrella/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.5.1"
+  changes:
+    - description: Update to readmen file
```
[the remark this answers is not in the document]
Comment (cherryleaf) [reply packages/cisco_umbrella/changelog.yml]: Done

**Comment by cherryleaf** [comment]
I think I've made the changes

**Comment by cherryleaf** [comment]
Cannot resolve

**Pull request closed without merging**